include/aspect/mesh_refinement/interface.h

```diff
@@ -222,6 +222,18 @@ namespace aspect
         void
         parse_parameters (ParameterHandler &prm);
 
+        /**
+         * Go through the list of all mesh refinement strategies that have been
+         * selected in the input file (and are consequently currently active)
+         * and see if one of them has the desired type specified by the
+         * template argument. If so, return a pointer to it. If no mesh
+         * refinement strategy is active that matches the given type, return a
+         * NULL pointer.
+         */
+        template <typename MeshRefinementType>
+        MeshRefinementType *
+        find_mesh_refinement_strategy () const;
```

Comment:
I would prefer if we would avoid this interface, and instead use an approach similar to the new `has_matching_postprocessor` and `get_matching_postprocessor` functions in the postprocess/interface.h file. I do not like returning null pointers, and we deprecated this interface for the postprocessors.